tools/cpp/tool_wrappers/mac/osx_clang_wrapper.sh

```diff
@@ -42,7 +42,7 @@ for i in "$@"; do
         LIBS="${BASH_REMATCH[1]} $LIBS"
     elif [[ "$i" =~ ^-L(.*)$ ]]; then
         LIB_DIRS="${BASH_REMATCH[1]} $LIB_DIRS"
-    elif [[ "$i" =~ ^-Wl,-rpath,\$ORIGIN/(.*)$ ]]; then
```

Comment:
This change moves the min OSX working version from 0.9.0 to 0.12.0, I probably pin a release for 0.9.0 to make this clear, I will also update the README to describe those changes.

Reply:
In all the articles I've been reading about `-rpath`, only ones about Linux versions were mentioning `$ORIGIN`, so it may well be a Linux-only feature. Apple-related ones seem to recommend using `@loader_path` or `@executable_path` with it.